production/deploy.sh

```diff
@@ -0,0 +1,37 @@
+# Deploy a CloudFormation Wazuh stack
+
+#!/bin/bash
+
+# Path of the parameters JSON file
+PARAMS_FILE='./parameters.json'
+
+# Path of the template file
+TEMPLATE_FILE='./wazuh_template.yml'
+
+# Stack name
+STACK_NAME='<cloudformation-stack-name>'
+
+# Bucket name
+BUCKET_NAME='<S3-bucket-name>'
+
+# If any file doesn't exist, then break the execution
+if ! [ -f "$PARAMS_FILE" ] || ! [ -f "$TEMPLATE_FILE" ]; then
+    echo "Missing template path or parameters file."
+    exit
+fi
+
+# Checking arguments
+if [ "$STACK_NAME" == "" ] || [ "$BUCKET_NAME" == "" ]; then
+    echo "Missing template path or parameters file."
+    exit
+fi
+
+# Uploading template to S3
+aws s3 cp $TEMPLATE_FILE s3://$BUCKET_NAME
+# Getting the template URL
+URL="https://<S3-bucket-name>.amazonaws.com/wazuh_template.yml"
```

Comment:
This Variable could use `${BUCKET_NAME}` to be rendered dynamically, right?

Reply:
Done in 3ad99aba96b01b0a6f9e944ba64c25d0c7c3a9c0. Thanks !